sdk/analytics/src/test/java/com/klaviyo/analytics/networking/KlaviyoApiClientTest.kt

```diff
@@ -100,14 +116,12 @@ internal class KlaviyoApiClientTest : BaseTest() {
 
     @Test
     fun `Enqueues a profile API call`() {
-        mockkStatic(Base64::class)
-        every { Base64.encodeToString(any(), any()) } returns "mock"
-
         assertEquals(0, KlaviyoApiClient.getQueueSize())
 
         KlaviyoApiClient.enqueueProfile(Profile().setAnonymousId(ANON_ID))
 
         assertEquals(1, KlaviyoApiClient.getQueueSize())
+        verify { logSpy.debug(any()) }
```

Comment:
Total nit: Possible to assert on something other than any.

Reply:
Again I did a bit of both, in this case the log output is pretty large, so i did the lazy thing of just using any

Reply:
I put logging into its own tests and tested for the actual output.